jobapi/middleware.go

```diff
@@ -0,0 +1,49 @@
+package jobapi
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+)
+
+func (s *Server) authMdlw(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer next.ServeHTTP(w, r)
```

Comment:
🤔 should we not call the next middleware in the stack if we're sending an unauthorised response?

Reply:
Yeah, probably not